pkg/profileparser/profileparser.go

```diff
@@ -840,7 +840,7 @@ func newStandardParser() *referenceParser {
 	if crherr != nil {
 		log.Error(crherr, "Could not register CIS Red Hat Linux reference parser") // not much we can do here..
 	}
-	nciperr := p.registerStandard("NERC-CIP", `^https://www\.nerc\.com/pa/Stand/Standard%20Purpose%20Statement%20DL/US_Standard_One-Stop-Shop\.xlsx$`)
+	nciperr := p.registerStandard("NERC-CIP", `^https://www\.nerc\.com/pa/Stand/AlignRep/One%20Stop%20Shop\.xlsx$`)
```

Comment:
I need to find a way to test these sort of changes, so we can catch them sooner.

Couple of ideas.

1. For each reference, call `HEAD $link` and make sure it doesn't return an `HTTP 404`
2. Find a rule that's used in all the OCP profiles and make sure it has the correct annotations (proving the parsing logic here worked)
3. Find a rule that's used in all the RHCOS profiles and make sure it has the correct annotations (providing the parsing logic here worked)